content/docs/2.1/concepts/authentication.md

```diff
@@ -223,3 +223,17 @@ You can tell KEDA to use Kiam via `podIdentity.provider`.
 podIdentity:
   provider: aws-kiam # Optional. Default: false
 ```
+
+## Global authentication with ClusterTriggerAuthentication
+
+For scalers where it makes sense to have global credentials you can use a `ClusterTriggerAuthentication`. All parameters work like with a normal `TriggerAuthentication` however Secrets will be looked up in the namespace KEDA is deployed in (`keda` by default).
+
+When referencing a `ClusterTriggerAuthentication` in a trigger, specify the Kind:
+
+```yaml
+    authenticationRef:
+      name: {trigger-authencation-name}
+      kind: ClusterTriggerAuthentication
+```
+
+The namespace to fetch secrets from can be overridden by setting a `$KEDA_CLUSTER_OBJECT_NAMESPACE` environment variable for the `keda-operator` container.
```

Comment:
Wouldn't it make sense to make this configurable in the ClusterTriggerAuthentication itself through something like this?

```yaml
secretTargetRef:                          # Optional.
  - parameter: connectionString           # Required - Defined by the scale trigger
    name: my-keda-secret-entity           # Required.
    key: azure-storage-connectionstring   # Required.
    namespace: <namespace-name-here>      # Optional.
```

Reply:
It would be good to add a section at the top that describes the different scopes before diving into it I think.

That way, it's not "hidden" at the bottom of the docs.

Reply:
Making this configurable per object makes me nervous about security implications. That could be used to read any secret in any namespace, possibly allowing for unexpected data access. Having it be limited to just one global namespace makes it easier for an admin to think through the risk factors.

Will rework the section order :)

Reply:
Makes sense.

Thanks!